molecule/Dockerfile.j2

```
@@ -17,11 +17,11 @@ RUN yum -y install java-11-openjdk \
       krb5-workstation \
       unzip
 
-RUN echo $'[Confluent.dist]\n\
-name=Confluent repository (dist)\n\
-baseurl=https://packages.confluent.io/rpm/6.2/8\n\
+RUN echo $'[Confluent.clients]\n\
+name=Confluent repository (clients)\n\
+baseurl=https://packages.confluent.io/clients/rpm/centos/$releasever/$basearch\n\
```

```suggestion
baseurl=https://packages.confluent.io/clients/rpm/centos/\$releasever/\$basearch\n\
```

Does this need to be escaped?